libs/accounts/recovery-phone/src/lib/recovery-phone.service.ts

```diff
@@ -49,4 +49,14 @@ export class RecoveryPhoneService {
     );
     return true;
   }
+
+  /**
+   * Remove phone number from an account. Each user can only have one associated
+   * phone number.
+   *
+   * @param uid
```

Comment:
Document what uid is. (This is just for consistency... we all know what it is ;p)